olp-cpp-sdk-core/tests/client/OlpClientTest.cpp

```diff
@@ -655,7 +656,8 @@ TEST_P(OlpClientTest, MethodDELETE) {
       });
 
   auto response = call_wrapper_->CallApi(
-      std::string(), "DELETE", std::multimap<std::string, std::string>(),
+      std::string(),
+      olp::http::NetworkRequest::HttpVerb::DEL, std::multimap<std::string, std::string>(),
```

Comment:
It is not related to your changes, but I'm curious why the `DELETE` verb was shortened to `DEL` - I think it would be better to leave it untouched to adhere with the RFC2616 specification.

Reply:
Yeah. I found that weird too. Will propose the appropriate change in a follow-up PR